src/panel/styled.ts

```diff
@@ -63,17 +63,22 @@ export const Body = styled.div`
   padding: 6px;
 `;
 
-export const Operation = styled.div`
+export const Operation = styled.div<{ selected: boolean }>`
   padding-left: 6px;
   padding: 3px;
   border-bottom: 1px solid #000;
   cursor: pointer;
   font-size: 18px;
   &:hover,
-  &:focus,
-  &:active {
```

Comment:
is there a reason why `:active` is removed?

Reply:
I checked, this CSS is for showing the div color when it is active, but actually it's not happening so I removed it. if you want I can keep it back.